<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphics_tab</class>
 <widget class="QWidget" name="graphics_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>20</width>
    <height>0</height>
   </size>
  </property>
  <property name="accessibleName">
   <string>graphics_tab</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_graphics_tab">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushbutton_residuals">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Residuals</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/timeline.svg</normaloff>../images/timeline.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushbutton_walltime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Simulation time</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/timeline.svg</normaloff>../images/timeline.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushbutton_vtk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>3d view</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/geometry.svg</normaloff>../images/geometry.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushbutton_mesh_stats">
     <property name="text">
      <string>Mesh Stats</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/bar_chart.svg</normaloff>../images/bar_chart.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushbutton_nit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Iterations (NIT)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/timeline.svg</normaloff>../images/timeline.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushbutton_dt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Time step (DT)</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/timeline.svg</normaloff>../images/timeline.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushbutton_monitors">
     <property name="text">
      <string>Monitors</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/timeline.svg</normaloff>../images/timeline.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color:None;</string>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="pushbutton_histogram">
     <property name="text">
      <string>Histogram</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/bar_chart.svg</normaloff>../images/bar_chart.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
